docs/iot/getting-started/setting-up-board.md

```diff
@@ -10,4 +10,4 @@ ARTIK530 or ARTIK530S | No | Tizen 5.0 | Linux | [ARTIK530 or ARTIK530S, no SD c
 ARTIK530 | Yes | Tizen 5.0 | Linux and Windows | [ARTIK530, SD card, Tizen 5.0](artik-sd-5.0.md)
 Raspberry Pi3 | Yes | Tizen 5.0 | Linux and Windows |  [Raspberry Pi3, Tizen 5.0](rpi3-5.0.md)
 
-For information on Tizen IoT Preview 2 (Legacy), see [Tizen IoT Preview 2 (Legacy).](iot_preview_2.md)
+For information on Tizen IoT Preview 2 (Legacy), see [Tizen IoT Preview 2.](../preview2/getting-started/overview.md)
```

Comment:
It is wired that this sentence suddenly pops out. In general, when there is a short explanation and then we use "For more information ...".

How about changing the sentence like below?
```
If you are interested in Tizen IoT Preview 2, see [Tizen IoT Preview 2.](../preview2/getting-started/overview.md)
```
or 

```
If you want to use Tizen IoT Preview 2, see [Tizen IoT Preview 2.](../preview2/getting-started/overview.md)
```

Reply:
agreed. I modify the commit now.